src/System.Data.SqlClient/pkg/System.Data.SqlClient.pkgproj

```diff
@@ -1,9 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
```

Comment:
Do we think it is better to put the version number in pkgproj or in the dir.props?

Reply:
Correct me if I'm wrong, but package version is only ever needed by the pkgproj, so it made more sense to me to scope it down to just the project that uses it. I think of dir.props as more general properties that apply to multiple projects. That said though, I'm open to changing it if you feel dir.props is a better place for it.

Reply:
I believe so and I'm fine with keeping the version here, just a thought when I saw the change.